### ISSUE TITLE
Import LilyPond 2.20 command set for autocompletion

### ISSUE THREAD

**Issue by sincere-music** [frescobaldi/frescobaldi#1292]
Updating the list of commands for autocompletion should be done with every stable release of LilyPond, IIUC. So finally now would be a good time.

**Comment by sincere-music** [comment]
I would volunteer to do it, but since I’m completely unfamiliar with Frescobaldi’s codebase or Python, it would be good to get some pointers (unless the task is more complex than I think it is and I lack qualification).

**Comment by wbsoft** [comment]
It is not complicated, and very welcome!
Just add the new commands to the `words.py` file in the Python-ly module: https://github.com/frescobaldi/python-ly/blob/master/ly/words.py
Please add "# since 2.20" at the end :-)

Then I'll release python-ly with the updated commands.

**Comment by uliska** [comment]
Actualky we *should* have a system to support both the current stable and developmret versions.

**Comment by wbsoft** [comment]
That's in the works. It is not easy to implement that with current `ly` module, but the upcoming `quickly` module will support it :-)

**Comment by sincere-music** [comment]
So how do the contents of
https://github.com/frescobaldi/frescobaldi/blob/master/frescobaldi_app/autocomplete/completiondata.py
relate to this? IIUC this imports the abovementioned `words.py`, but it would too have to be extended by new commands like RemoveAllEmptyStaves, wouldn’t it?

**Comment by wbsoft** [comment]
Yes, ly.words is mainly used for syntax highlighting, but the autocompleter can use it, too. You have to look carefully where to add it :-)

**Comment by sincere-music** [comment]
What this isn’t going to cover:
– auto-generated file(s): after noticing that `_lilypond_data.py` is generated from the LilyPond source, I stopped trying to add the new `font-features` property in the font-interface.
– syntax changes around keys and symbol/key lists like `\time 2,2,1 5/8` or `\tagGroup violin,oboe,bassoon` (see http://lilypond.org/doc/v2.20/Documentation/changes/index.html , section ‘New for Internal interfaces and functions’, 8th bullet point)

**Comment by sincere-music** [comment]
https://github.com/frescobaldi/python-ly/pull/140
